### PR TITLE
Update to version 6.0.0a3

### PR DESCRIPTION
**Pull request by dalthviz**
Dependencies changes:

- `ipython` >=8.12.2,<9.0.0,!=8.17.1
- `jedi` >=0.17.2,<0.20.0
- `pyqt` >=5.15,<5.16
- `pyqtwebengine` >=5.15,<5.16
- `python-lsp-server` >=1.9.0,<1.10.0
- `qdarkstyle` >=3.2.0,<3.3.0
- `qtawesome` >=1.3.0
- `qtconsole` >=5.5.0,<5.6.0
- `qtpy` >=2.4.0
- `spyder-kernels` >=3.0.0b3,<3.0.0b4
- `superqt` >=0.6.1,<1.0.0

Other changes:

- Remove constraint for `importlib-metadata` to workaround `flake8` issue with Python < 3.8

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dalthviz** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,8 +36,8 @@ requirements:
     - fcitx-qt5 >=1.2.7  # [linux64]
     - fzf >=0.42.0
     - intervaltree >=3.0.2
-    - ipython >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0,!=8.11.0,!=8.12.0,!=8.12.1
-    - jedi >=0.17.2,<0.19.0
+    - ipython >=8.13.0,<9.0.0,!=8.17.1
```
Comment (ccordoba12) [inline recipe/meta.yaml:39]:
```suggestion
    - ipython >=8.12.2,<9.0.0,!=8.17.1
```

The 8.12 series is the last one in IPython to support Python 3.8

**Review by ccordoba12 — COMMENTED** [review]
@dalthviz, I think my suggestion below should fix the error on the Python 3.8 slots.

In addition, please apply the change I did in PR https://github.com/conda-forge/spyder-feedstock/pull/151 to fix the issues we had in Mac M1 laptops with Python 3.8.

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz!

**Pull request merged**